tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -191,20 +163,26 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
         return true;
     }
 
-    if (vulkan_replay_options.Draw_Indices.size() && !vulkan_replay_options.RenderPass_Indices.size())
+    for (int i = 0; i < vulkan_replay_options.Draw_Indices.size(); i++)
     {
-        GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
-        GFXRECON_LOG_ERROR("DrawCall indices must always be acompanied by render pass indices");
-        return true;
-    }
+        // Make sure each Draw has corresponding RenderPass indices
+        if (vulkan_replay_options.Draw_Indices[i].size() != vulkan_replay_options.RenderPass_Indices[i].size())
```

Comment:
This check is not correct. There can be several draw calls between a BeginRenderpass and EndRenderpass.
I.e.
```
    "Draw": [
        [
            473,
            474,
            475,
            476,
            477,
            478,
            479,
            480,
            481,
            482,
            483,
            484,
            485,
            486
        ]
    ],
    "RenderPass": [
        [
            [
                466,
                487
            ]
        ]
    ],
```

Reply:
Fix in 2d8a134f8a9f952fd31a128e78d2b0b455f1a261